src/app/features/post/post.component.ts

```diff
@@ -113,9 +121,29 @@ export class PostComponent {
     private router: Router,
     private markdownService: MarkdownService,
     @Inject(DOCUMENT) private document: Document,
-    private htmlInMarkdownService: HtmlInMarkdownService
+    private htmlInMarkdownService: HtmlInMarkdownService,
+    private postUrlPipe: PostUrlPipe,
+    private meta: Meta
   ) {}
 
+  ngOnInit(): void {
+    this.post$
+      .pipe(
+        filter((value): value is Post => !!value),
+        map(value => this.postUrlPipe.transform(value, value.teaser)),
+        takeUntil(this.destroy$)
```

Comment:
Can we use [takeUntilDestroy](https://angular.io/api/core/rxjs-interop/takeUntilDestroyed)?

Reply:
It's outdated